packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -555,6 +563,7 @@ export const createRootSummarizerNodeWithGC = (
 		summarizeInternalFn,
 		config,
 		changeSequenceNumber,
+		"" /* summaryHandleId */,
```

Comment:
Never does incremental summary or what?

Reply:
Yeah, the root containerRuntime node never makes an incremental summary.